CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ All notable changes to this project will be documented in this file based on the
 * Add `user.full_name` field. #201
 * Add `network.community_id` field. #208
 * Add fields `geo.country_name` and `geo.region_iso_code`. #214
+* Add \`geo.name\` for ad hoc location names. #248
```

Comment:
Nit: Are the `\` on purpose?

Reply:
Oh no. That's a habit from code quoting field names in commit messages. Will remove ;-)